os/services/rpl-border-router/native/slip-dev.c

```diff
@@ -70,6 +70,18 @@ extern speed_t slip_config_b_rate;
 #define SEND_DELAY 0
 #endif
 
+/*  Depending on the slip radio device, sometimes packet drop exist
```

Comment:
Start the comment at the second line in the block:
```
/*
 * Depending on the slip radio device, sometimes packet drop exist
 *...
```

Reply:
Done